hack/ci/e2e-k8s.sh

```diff
@@ -64,8 +64,25 @@ build() {
 
 # up a cluster with kind
 create_cluster() {
-  # set KUBELET_LOG_FORMAT=structured to enable structured logging in kubelet
-  KUBELET_LOG_FORMAT="${KUBELET_LOG_FORMAT:-text}"
+  # Grab the version of the cluster we're about to start
+  KUBE_VERSION="$(docker run --rm --entrypoint=cat "kindest/node:latest" /kind/version)"
+
+  # Default Log level for all components in test clusters
+  KIND_CLUSTER_LOG_LEVEL=${KIND_CLUSTER_LOG_LEVEL:-4}
+
+  # potentially enable --logging-format
+  kubelet_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
+  if [ -n "${KUBELET_LOG_FORMAT:-}" ]; then
+    case "${KUBE_VERSION}" in
+     v1.1[0-8].*)
```

Comment:
how this plays with  versions on CI?

Reply:
well < 1.1X we've never supported.
this is actually basically same logic as the ga_only stuff below.

Reply:
and we only set KUBELET_LOG_FORMAT in 1.21 / HEAD CI in one job

Reply:
scratch it, is used below too, time to :zzz: